### PR TITLE
HY-2810 Release wf-common 1.1.3

### PR DESCRIPTION
**Pull request by robbecker-wf**
PRs contained in this release:

https://github.com/Workiva/wf-common/pull/49

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/wf-common/compare/1.1.2...Release_1.1.3

### CODE REVIEW

**Comment by lancefisher-wf** [comment]
+1

**Comment by patkujawa-wf** [comment]
+semver

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: dart
+sudo: false
```
Comment (robbecker-wf) [inline .travis.yml]: This somehow got Travis to pass the build.

**Comment by tomconnell-wf** [comment]
QA +1

- [x] Dev +1 only needed for release.

Ready for merge and tag.

**Comment by lancefisher-wf** [comment]
+1

**Pull request merged**